python/ray/tests/test_resource_demand_scheduler.py

```diff
@@ -3260,6 +3276,16 @@ def test_info_string_verbose():
   0.9/1 accelerator_type:V100
   1.00GiB/12.00GiB memory
   0B/4.00GiB object_store_memory
+
```

Comment:
This is nice - i wonder do we also have any e2e testing that tests the new node activity output? Like when the node activity data is not being mocked.